### ISSUE TITLE
PyLaGriT: move to separate repo

### ISSUE THREAD

**Issue by daniellivingston** [lanl/LaGriT#218]
PyLaGriT should probably be moved to its own repo so that:

- it can be installed with `pip` and `conda`
  - See [here](https://realpython.com/pypi-publish-python-package/)
- PyLaGriT can have a separate commit tree from LaGriT
- PyLaGriT can have a separate documentation URL (`lanl.github.io/PyLaGriT`)
- PyLaGriT can have its own continuous integration pipeline
  - No sense in LaGriT travis cases failing if it's a PyLaGriT bug, or vice versa

**Comment by daniellivingston** [comment]
Also, TINerator is going to be removed from the LaGriT repo into its own shortly-ish.

**Comment by daniellivingston** [comment]
@dharp thoughts?

**Comment by dharp** [comment]
I agree, at this point, this would be a good move. I guess the tricky part is ensuring that the pylagrit version someone installs with pip is consistent with their lagrit version. I think in most cases, this probably won't be an issue since lagrit command syntax has been pretty static during the pylagrit development years, but something to consider.
